libdnf/dnf-context.cpp

```diff
@@ -120,6 +123,7 @@ typedef struct
     gchar            *user_agent;
     gchar            *arch;
     guint            cache_age;     /*seconds*/
+    bool             cacheOnly{false};
```

Comment:
This probably needs to be `gboolean` since this is part of the glibified bit of libdnf.